examples/outbound_redis/app.py

```diff
@@ -38,4 +19,4 @@ def handle_request(request):
 
     return Response(200,
                     [("content-type", "text/plain")],
-                    bytes(f"Hello from Python! Got request URI: {request.uri}", "utf-8"))
+                    bytes(f"Executed outbound Redis commands: {request.uri}", "utf-8"))
```

Comment:
I'd suggest leaving the `request.uri` out, since it has nothing to do with the rest of the sentence.
```suggestion
                    bytes(f"Executed outbound Redis commands", "utf-8"))
```